src/components/HelpResources/HelpResources.test.js

```diff
@@ -8,8 +8,12 @@ jest.setTimeout(30000);
 
 describe("test the Help & Resources page", () => {
```

Comment:
When you merge the two components, just make sure to include the accessibility test.